sql-statements/sql-statement-explain-analyze.md

```diff
@@ -231,16 +231,16 @@ build_hash_table:{total:146.071334ms, fetch:110.338509ms, build:35.732825ms}, pr
 在 TiFlash 节点上执行的 `TableFullScan` 算子包含以下执行信息：
 
 ```sql
-tiflash_scan:{dmfile:{total_scanned_packs:2, total_skipped_packs:1, total_scanned_rows:16000, total_skipped_rows:8192, total_rough_set_index_load_time: 2ms, total_read_time: 20ms}, total_create_snapshot_time: 1ms}
+tiflash_scan:{dtfile:{total_scanned_packs:2, total_skipped_packs:1, total_scanned_rows:16000, total_skipped_rows:8192, total_rough_set_index_load_time: 2ms, total_read_time: 20ms}, total_create_snapshot_time: 1ms}
 ```
 
-+ dmfile：扫表过程中跟 dmfile 相关的执行信息
-    - `total_scanned_packs`：扫表过程中 dmfiles 中被扫描过的 pack 的数量
-    - `total_skipped_packs`：扫表过程中 dmfiles 中被跳过的 pack 的数量
-    - `total_scanned_rows`：扫表过程中 dmfiles 中被扫描过的 row 的数量
-    - `total_skipped_rows`：扫表过程中 dmfiles 中被跳过的 row 的数量 
-    - `total_rough_set_index_load_time`：扫表过程中相关 dmfiles 加载粗糙索引的总耗时
-    - `total_read_time`：扫表过程中读取 dmfiles 内数据的总耗时
++ `dtfile`：扫表过程中跟 dtfile 相关的执行信息
+    - `total_scanned_packs`：扫表过程中 dtfiles 中被扫描过的 pack 的数量
+    - `total_skipped_packs`：扫表过程中 dtfiles 中被跳过的 pack 的数量
+    - `total_scanned_rows`：扫表过程中 dtfiles 中被扫描过的 row 的数量
+    - `total_skipped_rows`：扫表过程中 dtfiles 中被跳过的 row 的数量 
+    - `total_rs_index_load_time`：扫表过程中相关 dtfiles 加载粗糙索引的总耗时
+    - `total_read_time`：扫表过程中读取 dtfiles 内数据的总耗时
```

Comment:
Some users want to have a deeper understanding about our detailed information. I enriched a bit so that it could be more understandable for these advanced users:

```suggestion
+ `dtfile`：扫表过程中与 DTFile 相关的信息；这基本对应了 Stable 层数据的读取情况
    - `total_scanned_packs`：DTFile 内累计读取的 Pack 的数量
    - `total_skipped_packs`：DTFile 内累计跳过的 Pack 的数量
    - `total_scanned_rows`：DTFile 内累计读取的行数（若存在多版本更新或删除，则每个版本独立计数）
    - `total_skipped_rows`：DTFile 内累计跳过的行数
    - `total_rs_index_load_time`：读取 DTFile 粗糙索引的累计耗时
    - `total_read_time`：读取 DTFile 数据的累计耗时
```

Reply:
@hongyunyan I also have a question, for "skipped packs", how is it counted? For example, may be we have range clips and RS masks, does it only count for RS masks? (i.e. skip == skip by RS index)